server/src/main/java/org/elasticsearch/discovery/PeerFinder.java

```diff
@@ -66,7 +66,8 @@ public abstract class PeerFinder {
         "discovery.request_peers_timeout",
         TimeValue.timeValueMillis(3000),
         TimeValue.timeValueMillis(1),
-        Setting.Property.NodeScope
+        Setting.Property.NodeScope,
+        Setting.Property.Deprecated
```

Comment:
This is a deprecation and strictly speaking behaviour change. Do we want to use a different label other than `>non-issue`?